openff/interchange/interop/gromacs/models/models.py

```diff
@@ -341,8 +341,8 @@ class GROMACSSystem(_BaseModel):
         dict(),
         description="Molecule types, keyed by name.",
     )
-    molecules: dict[str, int] = Field(
-        dict(),
+    molecules: list[tuple[str, int]] = Field(
+        list(),
         description="The number of each molecule type in the system, keyed by the name of each molecule.",
```

Comment:
(not blocking) "keyed" doesn't work any more since it's not a dict, just removing it should be fine. 
```suggestion
        description="The number of each molecule type in the system.",
```